prime-router/src/main/kotlin/cli/ValidateTranslationSchemaCommand.kt

```diff
@@ -0,0 +1,73 @@
+package gov.cdc.prime.router.cli
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.enum
+import gov.cdc.prime.router.azure.BlobAccess
+import gov.cdc.prime.router.fhirengine.translation.TranslationSchemaManager
+import org.apache.logging.log4j.kotlin.Logging
+
+class ValidateTranslationSchemaCommand :
+    CliktCommand(
+    name = "validateSchemas",
+    help = "Commands to validate translation schemas"
+),
+    Logging {
+
+    private val translationSchemaManager = TranslationSchemaManager()
+
+    /**
+     * The schema type to validate
+     */
+    private val schemaType: TranslationSchemaManager.SchemaType by option(
+        "--schema-type",
+        "-s",
+        help = "which schema type to process"
+    ).enum<TranslationSchemaManager.SchemaType>().required()
+
+    private val blobStoreConnection: String by option(
+        "--blob-store-connect",
+        "-c",
+        help = "The blob store connection string where schemas will be validated"
+    ).required()
+
+    private val blobStoreContainer: String by option(
+        "--blob-store-container",
+        "-b",
+        help = "The container in azure where the schemas are stored"
+    ).required()
+
+    override fun run() {
+        val validationResults =
+            translationSchemaManager.validateManagedSchemas(
+                schemaType,
+                BlobAccess.BlobContainerMetadata(blobStoreContainer, blobStoreConnection)
+            )
+        if (validationResults.size > 0) {
+            val allPassed = validationResults.all { it.passes }
+
+            val individualResults = validationResults.map { result ->
+                """
+            Path validated: ${result.path}
```

Comment:
Another set of confusing indentation. Is this one linter related too?

It's hard to see that this is part of the body of the `map` function at first glance.

Reply:
Yea, this is how the linter set it up, but I'll see if I can massage it a bit more